### ISSUE TITLE
Convert to black and white: on by default

### ISSUE THREAD

**Issue by tomworthington** [ayastreb/bandwidth-hero-proxy#4]
"Convert to black and white" is on by default. I suggest changing this to "off", as few people will want their web pages in monochrome. Apart from that it is [good](http://blog.tomw.net.au/2017/11/reducing-dowload-traffic-for-mobile.html).

**Comment by ttpro1995** [comment]
Yeah, It take me a while to find the option to change it "off" :v

**Comment by swordgreen** [comment]
@ttpro1995 Where to change it?

**Comment by swordgreen** [comment]
@ttpro1995 I found it. It's in the Firefox addon. I was looking for it in the source code on the server-side.

**Comment by ttpro1995** [comment]
The option to change Black and White on/off is on the extension setting
